### PR TITLE
Complete the target setting of export jar

### PR DESCRIPTION
**Pull request by CsCherrYY**
#362

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ A lightweight extension to provide additional Java project explorer features. It
 | `java.dependency.autoRefresh` | Specify whether to automatically sync the change from editor to the Java Projects explorer. | `true` |
 | `java.dependency.refreshDelay` | The delay time (ms) the auto refresh is invoked when changes are detected. | `2000ms` |
 | `java.dependency.packagePresentation` | Specify how to display the package. Supported values are: `flat`, `hierarchical`.| `flat` |
-| `java.project.exportJar.targetPath` | The default output path of export jar. Supported values are: `${workspaceFolder}/${workspaceFolderBasename}.jar`, `askUser`.| `${workspaceFolder}/${workspaceFolderBasename}.jar` |
+| `java.project.exportJar.targetPath` | The output path of export jar. When this setting is empty or equals `askUser`, a dialog will appear to ask user when export jar.| `${workspaceFolder}/${workspaceFolderBasename}.jar` |
```
Comment (jdneo) [inline README.md:40]: When this setting is empty or equals `askUser`, a file explorer will pop up to let the user select the output location.

File: package.nls.json [hunk 2]
```diff
@@ -24,5 +24,5 @@
   "configuration.java.dependency.autoRefresh": "Synchronize Java Projects explorer with changes",
   "configuration.java.dependency.refreshDelay": "The delay time (ms) the auto refresh is invoked when changes are detected",
   "configuration.java.dependency.packagePresentation": "Package presentation mode: flat or hierarchical",
-  "configuration.java.project.exportJar.targetPath": "The default output path of export jar"
+  "configuration.java.project.exportJar.targetPath": "Specifies the output path of export jar. The default value is `${workspaceFolder}/${workspaceFolderBasename}.jar`. When it is empty or equals `askUser`, you will specify the output path manually. **Note**: Since `askUser` will be deprecated in the future, it's recommended to simply set this setting to empty instead."
```
Comment (jdneo) [inline package.nls.json:27]: To select the output location manually each time exporting the jar file. Simply leave it empty or set it to `askUser`.

Comment (jdneo) [inline package.nls.json:27]: **empty**

File: package.nls.zh.json [hunk 3]
```diff
@@ -24,5 +24,5 @@
   "configuration.java.dependency.autoRefresh": "在 Java 项目管理器中自动同步修改",
   "configuration.java.dependency.refreshDelay": "控制 Java 项目管理器刷新的延迟时间 (毫秒)",
   "configuration.java.dependency.packagePresentation": "Java 包显示方式: 平行显示或者分层显示",
-  "configuration.java.project.exportJar.targetPath": "导出 Jar 文件的默认路径, 默认值为 `${workspaceFolder}/${workspaceFolderBasename}.jar` .当此设置为空或 `askUser` 时, 您将会在导出 jar 文件时手动选择输出目录. **Note**: 当您想要手动选择输出目录时, 我们建议将此值置为空, 因为`askUser`将来会被弃用."
+  "configuration.java.project.exportJar.targetPath": "导出 Jar 文件的默认路径, 默认值为 `${workspaceFolder}/${workspaceFolderBasename}.jar` .为了在导出 jar 文件时手动选择输出目录, 您可以当此选项置为**空**或 `askUser` ."
```
Comment (jdneo) [inline package.nls.zh.json:27]: 此句有歧义：您可以当此选项置为**空**或 `askUser`

**Review by jdneo — CHANGES_REQUESTED** [review]

File: package.nls.json [hunk 4]
```diff
@@ -24,5 +24,5 @@
   "configuration.java.dependency.autoRefresh": "Synchronize Java Projects explorer with changes",
   "configuration.java.dependency.refreshDelay": "The delay time (ms) the auto refresh is invoked when changes are detected",
   "configuration.java.dependency.packagePresentation": "Package presentation mode: flat or hierarchical",
-  "configuration.java.project.exportJar.targetPath": "The default output path of export jar"
+  "configuration.java.project.exportJar.targetPath": "Specifies the output path of export jar. The default value is `${workspaceFolder}/${workspaceFolderBasename}.jar`. To select the output location manually each time exporting the jar file. Simply leave it **empty** or set it to `askUser`."
```
Comment (jdneo) [inline package.nls.json:27]: Simply leave it as `""` or set it to `"askUser"`

File: package.nls.zh.json [hunk 5]
```diff
@@ -24,5 +24,5 @@
   "configuration.java.dependency.autoRefresh": "在 Java 项目管理器中自动同步修改",
   "configuration.java.dependency.refreshDelay": "控制 Java 项目管理器刷新的延迟时间 (毫秒)",
   "configuration.java.dependency.packagePresentation": "Java 包显示方式: 平行显示或者分层显示",
-  "configuration.java.project.exportJar.targetPath": "导出 Jar 文件的默认路径"
+  "configuration.java.project.exportJar.targetPath": "导出 Jar 文件的默认路径, 默认值为 `\"${workspaceFolder}/${workspaceFolderBasename}.jar\"` .为了在导出 jar 文件时手动选择输出目录, 您可以将此选项置为`\"\"`或 `\"askUser\"` ."
```
Comment (jdneo) [inline package.nls.zh.json:27]: Don't use `.` in Chinese sentence

**Review by jdneo — CHANGES_REQUESTED** [review]

File: package.nls.zh.json [hunk 6]
```diff
@@ -24,7 +24,9 @@
   "configuration.java.dependency.autoRefresh": "在 Java 项目管理器中自动同步修改",
   "configuration.java.dependency.refreshDelay": "控制 Java 项目管理器刷新的延迟时间 (毫秒)",
   "configuration.java.dependency.packagePresentation": "Java 包显示方式: 平行显示或者分层显示",
-  "configuration.java.project.exportJar.targetPath": "导出 Jar 文件的默认路径，默认值为 `\"${workspaceFolder}/${workspaceFolderBasename}.jar\"`。为了在导出 jar 文件时手动选择输出目录，您可以将此选项置为`\"\"`或 `\"askUser\"`。",
+  "configuration.java.project.exportJar.targetPath": "导出 Jar 文件的默认路径，默认值为 `${workspaceFolder}/${workspaceFolderBasename}.jar` 。为了在每次导出 jar 文件时手动选择输出目录，您可以将此选项置为空串或 `askUser` 。",
+  "configuration.java.project.exportJar.targetPath.workspaceFolder": "导出 Jar 文件到工作空间文件夹下。 Jar 文件的名称和工作空间文件夹的名称相同。",
```
Comment (jdneo) [inline package.nls.zh.json:28]: `下。 Jar ` extra spaces between `。` & `jar`

File: package.nls.zh.json [hunk 7]
```diff
@@ -24,7 +24,9 @@
   "configuration.java.dependency.autoRefresh": "在 Java 项目管理器中自动同步修改",
   "configuration.java.dependency.refreshDelay": "控制 Java 项目管理器刷新的延迟时间 (毫秒)",
   "configuration.java.dependency.packagePresentation": "Java 包显示方式: 平行显示或者分层显示",
-  "configuration.java.project.exportJar.targetPath": "导出 Jar 文件的默认路径，默认值为 `\"${workspaceFolder}/${workspaceFolderBasename}.jar\"`。为了在导出 jar 文件时手动选择输出目录，您可以将此选项置为`\"\"`或 `\"askUser\"`。",
+  "configuration.java.project.exportJar.targetPath": "导出 Jar 文件的默认路径，默认值为 `${workspaceFolder}/${workspaceFolderBasename}.jar` 。为了在每次导出 jar 文件时手动选择输出目录，您可以将此选项置为空串或 `askUser` 。",
```
Comment (jdneo) [inline package.nls.zh.json:27]: 为了在每次导出 jar 文件时手动选择输出目录，您也可以将此选项置为空串或 `askUser` 来手动选择 `jar` 文件的导出目录。

**Review by jdneo — CHANGES_REQUESTED** [review]

File: package.nls.zh.json (hunk 7 above)
Comment (CsCherrYY) [reply package.nls.zh.json:27]: How about "您也可以将此选项置为空串或 `askUser` 来在每次导出 jar 文件时手动选择导出目录" ?

Comment (jdneo) [reply package.nls.zh.json:27]: sounds good

**Review by jdneo — APPROVED** [review]

**Pull request merged**